src/software/thunderscope/binary_context_managers/tigers_autoref.py

```diff
@@ -46,18 +45,17 @@ def __init__(
         tick_rate_ms: int,
         ci_mode: bool = False,
         buffer_size: int = 5,
-        supress_logs: bool = True,
+        suppress_logs: bool = True,
         show_gui: bool = False,
     ) -> None:
-        """
-        Constructor
+        """Constructor
 
         :param gc:              gamecontroller instance
         :param tick_rate_ms:    the interval between subsequent SSL Vision packets
         :param ci_mode:         true to run the autoref binary in CI mode (use system time or use the time from the
                                 CiInput messages)
         :param buffer_size:     buffer size for the SSL wrapper and referee packets
-        :supress_logs:          true silences logs from the Autoref binary, otherwise shows them (its very verbose)
+        :suppress_logs:          true silences logs from the Autoref binary, otherwise shows them (its very verbose)
         :show_gui:              true shows the Tigers' autoref GUI, false runs it in headless mode
```

Comment:
```suggestion
        :param suppress_logs:          true silences logs from the Autoref binary, otherwise shows them (its very verbose)
        :param show_gui:              true shows the Tigers' autoref GUI, false runs it in headless mode
```